app/src/controllers/sourceSystemKind.ts

```diff
@@ -0,0 +1,16 @@
+import { sourceSystemKindService } from '../services';
+
+import type { Request, Response, NextFunction } from 'express';
+
+const controller = {
+  getSourceSystemkinds: async (req: Request, res: Response, next: NextFunction) => {
```

Comment:
kind is a separate word, should be `getSourceSystemKinds`
This should also be changed in service, and yaml file for `operationId`